### ISSUE TITLE
Brute Force Detection on Application level not normal behavior #8583

### ISSUE THREAD

**Issue by ppgravitee** [gravitee-io/issues#8593]
<p>h2\. Describe the bug (<a href="https://github.com/gravitee-io/issues/issues/8583" title="smart-link" class="external-link" rel="nofollow noreferrer">https://github.com/gravitee-io/issues/issues/8583</a>)</p> 
<p><span class="image-wrap" style=""><img src="/secure/attachment/10054/10054_1f4a5.png" alt="boom"></span></p> 
<p>We have a domain with two applications. Each of the two apps has Brute Force Detection configuration.<br> There is a strange behavior during the connection and blocking of account on the two applications (A and B).</p> 
<p>h2\. To Reproduce</p> 
<p><span class="image-wrap" style=""><img src="/secure/attachment/10052/10052_1f304.png" alt="sunrise_over_mountains"></span></p> 
<p>Steps to reproduce the behavior:</p> 
<ol> 
 <li>User makes n failed login attempts on A (wrong password) with n greater than the number of attempts authorized by A's config =&gt; the user account is blocked for A(Settings -&gt; User).</li> 
 <li>User now tried to login to application B and he passed.</li> 
 <li>After step 2\. the user is no longer blocked in the Settings -&gt; Users</li> 
 <li>After step 2., he can no longer unlock his account from A, whether by the unlock link or by password reset =&gt; step 2 changes the profile of the user who becomes associated with application B.</li> 
 <li>Changing password from B does not unblock user on A</li> 
 <li>Changing the app associated with the user in their profile back to A makes the unlock work</li> 
</ol> 
<p>This behavior is not normal. Users are managed at the domain level. How can we consider a user blocked for one app and not for the other when it's the same user! we end up with a user who is blocked on the mobile app but he can still connect to the web application.</p> 
<p>h2\. Expected behavior</p> 
<p><span class="image-wrap" style=""><img src="/secure/attachment/10053/10053_1f308.png" alt="rainbow"></span></p> 
<p>If the user (domain level) is blocked on one application they should be blocked for all applications.</p> 
<p>h2\. Desktop:</p> 
<p><span class="image-wrap" style=""><img src="/secure/attachment/10055/10055_1f4bb.png" alt="computer"></span></p> 
<p><b><em>Please complete the following information:</em></b></p> 
<ul> 
 <li>Environment: Docker-Desktop 3.17.5\. 3.18.5</li> 
 <li>OS: Windows 11</li> 
 <li>Browser: Google Chrome</li> 
</ul> 
<p>h2\. Potential impacts</p> 
<p><span class="image-wrap" style=""><img src="/secure/attachment/10051/10051_26a0.png" alt="warning"></span></p> 
<p><b><em>What are the impacted versions?</em></b></p> 
<ul> 
 <li>3.17.5, 3.18.5</li> 
</ul>

**Comment by leleueri** [comment]
duplicate of #8583
